{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py

```diff
@@ -35,35 +35,42 @@ def get_tm_session(session_factory, transaction_manager, request=None):
     - When using scripts you should wrap the session in a manager yourself.
       For example::
 
-          import transaction
+    .. code-block:: python
 
-          engine = get_engine(settings)
-          session_factory = get_session_factory(engine)
-          with transaction.manager:
-              dbsession = get_tm_session(session_factory, transaction.manager)
+        import transaction
+
+        engine = get_engine(settings)
+        session_factory = get_session_factory(engine)
+        with transaction.manager:
+            dbsession = get_tm_session(session_factory, transaction.manager)
 
     This function may be invoked with a ``request`` kwarg, such as when invoked
-    by the reified ``request.dbsession`` Pyramid Request attribute. The default
-    value, for backwards compatibility, is ``None``.
+    by the reified ``.dbsession`` Pyramid request attribute which is configured
+    via the ``includeme`` function below. The default value, for backwards
+    compatibility, is ``None``.
+
+    The ``request`` kwarg is used to populate the ``sqlalchemy.orm.Session``'s
+    "info" dict.  The "info" dict is the official namespace for developers to
+    stash session-specific information.  For more information, please see the
+    SQLAlchemy docs:
+    https://docs.sqlalchemy.org/en/13/orm/session_api.html?#sqlalchemy.orm.session.Session.params.info
```

Comment:
This redirects to last stable released version:

```suggestion
    https://docs.sqlalchemy.org/en/stable/orm/session_api.html?#sqlalchemy.orm.session.Session.params.info
```

Or the latest (under development) version:

https://docs.sqlalchemy.org/en/latest/orm/session_api.html?#sqlalchemy.orm.session.Session.params.info

I'd go with stable.

Reply:
I mean, rather than specifying a specific version number in the URL, because that will change as soon as the development version is released as stable.